Library/Homebrew/brew.sh

```diff
@@ -663,15 +676,16 @@ then
   #
   # Don't need shellcheck to follow this `source`.
   # shellcheck disable=SC1090
-  source "$HOMEBREW_BASH_COMMAND"
-  { update-preinstall "$@"; "homebrew-$HOMEBREW_COMMAND" "$@"; exit $?; }
+  source "${HOMEBREW_BASH_COMMAND}"
+  { update-preinstall "$@"; "homebrew-${HOMEBREW_COMMAND}" "$@"; exit $?; }
 else
   # Don't need shellcheck to follow this `source`.
   # shellcheck disable=SC1090
-  source "$HOMEBREW_LIBRARY/Homebrew/utils/ruby.sh"
+  source "${HOMEBREW_LIBRARY}/Homebrew/utils/ruby.sh"
   setup-ruby-path
 
   # Unshift command back into argument list (unless argument list was empty).
-  [[ "$HOMEBREW_ARG_COUNT" -gt 0 ]] && set -- "$HOMEBREW_COMMAND" "$@"
-  { update-preinstall "$@"; exec "$HOMEBREW_RUBY_PATH" $HOMEBREW_RUBY_WARNINGS "$RUBY_DISABLE_OPTIONS" "$HOMEBREW_LIBRARY/Homebrew/brew.rb" "$@"; }
+  [[ "${HOMEBREW_ARG_COUNT}" -gt 0 ]] && set -- "${HOMEBREW_COMMAND}" "$@"
+  # shellcheck disable=SC2154
```

Comment:
Which variable needs this?

Reply:
HOMEBREW_RUBY_PATH needs it.
```
  { update-preinstall "$@"; exec "${HOMEBREW_RUBY_PATH}" "${HOMEBREW_RUBY_WARNINGS}" "${RUBY_DISABLE_OPTIONS}" "${HOMEBREW_LIBRARY}/Homebrew/brew.rb" "$@"; }
                                  ^-------------------^ SC2154: HOMEBREW_RUBY_PATH is referenced but not assigned.
```

Reply:
Gotcha, thanks. Comment saying it's set by `ruby.sh` would be good.